app/controllers/shopify_app/extension_verification_controller.rb

```diff
@@ -2,19 +2,12 @@
 
 module ShopifyApp
   class ExtensionVerificationController < ActionController::Base
+    include ShopifyApp::PayloadVerification
     protect_from_forgery with: :null_session
     before_action :verify_request
 
-    private
```

Comment:
I think we should keep the `private` declaration in the extracted concern. In a controller, typically only the action methods should be public.